app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -2,52 +2,38 @@ import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
 import ReactMarkdown from "react-markdown";
 import { connectStateResults } from "react-instantsearch/connectors";
+import { Props as GoogleMapProps, Coords } from "google-map-react";
 import { whiteLabel } from "utils";
 import { SearchMap } from "components/search/SearchMap/SearchMap";
 import ResultsPagination from "components/search/Pagination/ResultsPagination";
 import { Texting } from "components/Texting";
 import { ClinicianActions } from "components/ucsf/ClinicianActions/ClinicianActions";
 import { ClientHandouts } from "components/ui/ClientHandoutsModal/ClientHandouts";
+import { SearchHit, transformHits } from "../../../models/SearchHits";
 import { icon } from "../../../assets";
-import { parseAlgoliaSchedule } from "../../../models";
 import styles from "./SearchResults.module.scss";
 
-/**
- * Transform Algolia search hits such that each hit has a recurringSchedule that
- * uses the time helper classes.
- */
-const transformHits = (hits) =>
-  hits.map((hit) => {
-    const inheritedSchedule =
-      hit.schedule && hit.schedule.length
-        ? hit.schedule
-        : hit.resource_schedule;
-    const recurringSchedule =
-      inheritedSchedule && inheritedSchedule.length
-        ? parseAlgoliaSchedule(inheritedSchedule)
-        : null;
-    return { ...hit, recurringSchedule };
-  });
-
-// Todo: setExpandList will be used as part of next stage of multiple location work
-// eslint-disable-next-line no-unused-vars
-const SearchResults = ({ searchResults, expandList, setExpandList }) => {
-  const [centerCoords, setCenterCoords] = useState(null);
-  const [googleMapObject, setMapObject] = useState(null);
+const SearchResults = ({ searchResults, expandList }) => {
```

Comment:
Could you declare the type of the props for this component? According to `@types/react-instantsearch-core`, the [`connectStateResults()`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f33d088a06faa38729454ae4278624bd84441b35/types/react-instantsearch-core/index.d.ts#L597) higher-order component should inject a [`StateResultsProvided`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f33d088a06faa38729454ae4278624bd84441b35/types/react-instantsearch-core/index.d.ts#L571C29-L571C29) as the props. The `TDoc` type parameter looks like it ultimately gets used to describe the type of the hits in [`SearchResults.hits`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f33d088a06faa38729454ae4278624bd84441b35/types/react-instantsearch-core/index.d.ts#L710C18-L710C18), so I think it might work to pass in `SearchHit` as that type parameter.

Reply:
Ah this was also throwing me for a loop. Ultimately, I used the actual `SearchResults` class, which I aliased as `SearchResultsProps`